demostore_automation/tests/frontend/my_account/test_login_non_existing_user.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from demostore_automation.src.pages.MyAccountSignedOutPage import MyAccountSignedOutPage
+
+pytestmark = [pytest.mark.fe, pytest.mark.regression, pytest.mark.smoke, pytest.mark.my_account]
+
+@pytest.mark.usefixtures("init_driver")
+class TestLoginNegative:
+    
```

Comment:
just formatting. When you add doc-string put it right under the class/function. do not add space. So remove the space above the doc-string. Good job adding doc string tho.